### ISSUE TITLE
Could you please write steps how to run this code? 

### ISSUE THREAD

**Issue by ghost** [briansjavablog/spring-webservices-tutorial#1]
Could you please write steps how to run this code? Please help us for learner

**Comment by HawiCaesar** [comment]
Yes please...and is it possible to have a basic security example as well?

**Comment by vij658** [comment]
It would be great to list down the steps to run this code. Can you please include them.

**Comment by pskoli** [comment]
From your ide Run maven build, that should take care of everything
